### PR TITLE
User-friendly review

### PR DESCRIPTION
**Pull request by FranFin**
A lot of stuff has been changed.

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -10,18 +10,19 @@ Getting Started
 Installation
 ^^^^^^^^^^^^
 
-To build `integration-services` it is recommended use a
-`colcon workspace <https://colcon.readthedocs.io/en/released/user/quick-start.html>`__.
-The `integration-services` repo consists of many cmake packages which can be configured and built manually,
-but colcon makes the job much smoother.
+..
+ To build :code:`integration-service`, we recommend to use a `colcon workspace <https://colcon.readthedocs.io/en/released/user/quick-start.html>`__.
```
[the remark this answers is not in the document]
Comment (FranFin) [reply docs/getting_started.rst:14]: ok, deleted.

File: docs/getting_started.rst [hunk 2]
```diff
@@ -30,15 +31,18 @@ Create a `colcon workspace <https://colcon.readthedocs.io/en/released/user/quick
     cd workspaces/is
     git clone ssh://git@github.com/eProsima/SOSS-DDS src/is -b feature/xtypes-dds
 
-Once `integration-service` is in the `src` directory of your colcon workspace and you have sourced `soss` you can run
-`colcon build`:
+**Note**: if you don't have :code:`Fast-RTPS` installed, you can add the :code:`--recursive` flag to the
+:code:`git clone` command as an alternative to download the required :code:`Fast-RTPS` third-parties.
+
+Once :code:`integration-service` is in the `src` directory of your colcon workspace and you have sourced
+:code:`soss` as explained in REF., you can run :code:`colcon build`:
```
[the remark this answers is not in the document]
Comment (FranFin) [reply docs/getting_started.rst:38]: changed:
in REF.
to:
here [TODO: add link to Getting Started section of soss documentation]

File: docs/use_cases.rst [hunk 3]
```diff
@@ -24,43 +31,56 @@ See :ref:`ROS2` for an example of this use-case.
...
+
+Once all the protocols of interest are communicated with :code:`integration-service`,
+each via a dedicated **System-Handle**, the inter-components communication
+can be easily implemented by means of a YAML configuration file, rather than by creating a dedicated 
+bridge for each pair of communicating components.
+For a system made of *N* components, this means that the number of new software parts to add grows as *N*
+rather than *N²*.
+
+Notice that :code:`integration-service` already provides the **System-Handle** for most of the more common
+protocols.
```
[the remark this answers is not in the document]
Comment (FranFin) [reply docs/use_cases.rst:59]: changed to:
for some of the most common protocols.

File: docs/use_cases.rst [hunk 4]
```diff
@@ -24,43 +31,56 @@ See :ref:`ROS2` for an example of this use-case.
...
-For this example, a `Some/IP` **System-Handle** already exists, but if it doesn't, the user can create his own
-**System-Handle** and then, will be able to communicate with any other protocol already supported by
-`integration-service`, not only `DDS`.
+By using :code:`integration-service` instead, this communication can be achieved with minimum user's effort.
+In this specific case, a :code:`SOME/IP` **System-Handle** already exists, so the communication with :code:`DDS` is 
+essentially direct.
+However, the communication is straightforward enough even if a dedicated **System-Handle** doesn't exist yet, as
+the user can create his own **System-Handle**, thus becoming able to communicate with :code:`DDS` and
+any other protocol already supported by :code:`integration-service`.
+For more information regarding how to generate a **System-Handle** from scratch, please consult REF.
```
[the remark this answers is not in the document]
Comment (FranFin) [reply docs/use_cases.rst:83]: Changed to:
please consult the *System-Handle creation* section of the :code:`soss` documentation.

where System-Handle creation will be a link in the future.

File: docs/examples.rst [hunk 5]
```diff
@@ -49,26 +68,33 @@ command.
...
-This example integrates a `dds` application into a `Orion Context-Broker` system.
+This example integrates a :code:`DDS` application into a :code:`Orion Context-Broker` system.
+
+To make this example work, you will require:
 
-To work this example, it is required an accesible `contextBroker` service, and install `SOSS-FIWARE` along with
-`integration-service`. To feed the `contextBroker` the example will use a `Fast-RTPS` HelloWorld publisher,
-so this example is needed too.
+- An accesible :code:`contextBroker` service.
+- An installation of the :code:`SOSS-FIWARE` **System-Handle** (please refer to REF. for downloading the dedicated repo)
```
[the remark this answers is not in the document]
Comment (FranFin) [reply docs/examples.rst:76]: ahahaha. ok. changed.

File: docs/examples.rst [hunk 6]
```diff
@@ -49,26 +68,33 @@ command.
...
+- An installation of :code:`Fast-RTPS` (at least v1.9.2) with the *HelloWorld* example working. Indeed, in order to feed
+  the :code:`contextBroker`, the example will use a :code:`Fast-RTPS` HelloWorld *publisher*.
 
-The file `soss-dds/examples/udp/dds_fiware.yaml` must be edited to match the `contextBroker` configuration
+The file :code:`soss-dds/examples/udp/dds_fiware.yaml` must be edited to match the :code:`contextBroker` configuration
 in the testing environment.
```
[the remark this answers is not in the document]
Comment (FranFin) [reply docs/examples.rst:81]: added.

File: docs/examples.rst [hunk 7]
```diff
@@ -49,26 +68,33 @@ command.
...
 
-Open three terminals (replace <url> with the location of the `contextBroker`):
+**Note**: If you built the :code:`integration-service` and/or :code:`SOSS-FIWARE` packages with colcon, please make sure
+to have done all the required sourcing of the colcon overlays or, in alternative, to have added the opportune
+source commands to the .bashrc file, as explained in the :ref:`Getting Started <getting started>` section.
 
-- In the first terminal, execute the HelloWorld publisher:
+Open three terminals (replace <url> with the location of the :code:`contextBroker`):
```
[the remark this answers is not in the document]
Comment (FranFin) [reply docs/examples.rst:87]: added.

File: docs/examples.rst [hunk 8]
```diff
@@ -104,106 +133,122 @@ Open three terminals (replace <url> with the location of the `contextBroker`):
...
 
-This test will launch a `ROS 2` talker in the *server* machine, and a `ROS 2` listener in the *client* machine.
-In both machines, an `integration-service` instance will communicate with the other using WAN-TCP communication
-capabilities of `Fast-RTPS`.
+This test will launch a :code:`ROS2` talker in the *server* machine, and a :code:`ROS2` listener in the *client*
+machine. In both machines, an :code:`integration-service` instance will communicate with the other using WAN-TCP
+communication capabilities of `Fast-RTPS`.
 
-So, the requirements in both machines are:
+So, the requirement in both machines is to have :code:`ROS2`(Crystal or superior) installed
```
Comment (LuisGP) [inline docs/examples.rst:214]:
```suggestion
So, the requirement in both machines is to have :code:`ROS2` (Crystal or superior) installed
```

Comment (FranFin) [reply docs/examples.rst:214]: ok.

**Pull request merged**